src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -329,11 +330,32 @@ class ChatAddAction extends Action2 {
 	}
 }
 
+class ChatReleaseNotesAction extends Action2 {
+	constructor() {
+		super({
+			id: `workbench.action.chat.releaseNotes`,
+			title: localize2('chat.releaseNotes.label', "Explore New Features"),
+			category: CHAT_CATEGORY,
+			f1: false,
+			menu: {
+				id: MenuId.ViewTitle,
+				when: ContextKeyExpr.equals('view', ChatViewId)
+			},
+		});
+	}
+
+	async run(accessor: ServicesAccessor, ...args: any[]) {
+		const commandService = accessor.get(ICommandService);
+		return await commandService.executeCommand('update.showCurrentReleaseNotes');
+	}
+}
```

Comment:
If this is only for menus I think we can use `MenuRegistry.appendMenuItem` to do this without adding a new id.

Reply:
Good point